source/Recommendations.Core/Parsing/ParsingErrorReason.cs

```diff
@@ -34,11 +34,6 @@ public enum ParsingErrorReason
         /// </summary>
         ItemIdTooLong = 5,
 
-        /// <summary>
-        /// The item id string contains invalid characters.
-        /// </summary>
-        IllegalCharactersInItemId = 6,
```

Comment:
Please keep this in place for backward compatibility - existing models may contain this error reason as part of their satistics (Model->ModelStatistics->ParsingReport->LineParsingError->ParsingErrorReason). Removing this may cause deserialization errors.

Reply:
you may change the comment above it to denote it is now deprecated (adding a "[Deprecated]" prefix for example)

Reply:
Good point. Done!